docs/reference/build/scroll.js

```diff
@@ -65,7 +65,7 @@ $(document).ready(function() {
             currL1Nav = getNavNode(activeSection.token);
             currL1Nav.show('fast');
         } 
-        // If active active is not the same as previous, hide previous L1Nav and show current L1Nav; set previous to current
+        // If active is not the same as previous, hide previous L1Nav and show current L1Nav; set previous to current
```

Comment:
This will also be obliterated by a `kubebuilder docs` run. I believe the original file is [here](https://github.com/Birdrock/brodocs/blob/master/scroll.js) if you would like to fix this at the source.